org-roam.el

```diff
@@ -462,11 +463,12 @@ Examples:
                        target))
      description)))
 
-(defun org-roam-insert (prefix &optional filter-fn)
+(defun org-roam-insert (&optional prefix filter-fn description)
   "Find an Org-roam file, and insert a relative org link to it at point.
 If PREFIX, downcase the title before insertion.
 FILTER-FN is the name of a function to apply on the candidates
-which takes as its argument an alist of path-completions.  See
+which takes as its argument an alist of path-completions.
+If DESCRIPTION is provided, use this as the link label. See
```

Comment:
If `prefix` is meant to downcase the title before insertion, can we give it a descriptive name? e.g. `downcase` or `lowercase`?